### PR TITLE
Add a same origin warning to the base-url docs

### PR DESCRIPTION
**Pull request by illume**
Some documentation about how people should take care if using headlamp on a base-url with other apps.

### CODE REVIEW

File: docs/installation/base-url.md [hunk 1]
```diff
@@ -6,8 +6,15 @@ linkTitle: Base URL
...
 
-- default at the root of the domain: `http://headlamp.example.com/`.
-- base-url `http://example.com/headlamp/` 
+- default at the root of the domain: `https://headlamp.example.com/`.
+- base-url `https://example.com/headlamp/` 
+
+
+## A warning about multiple apps on the same sub domain
+
+Hosting multiple websites(apps) on the [same origin](https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy) can lead to possible conflicts between the apps. Each app is able to access information and make requests of the other. Therefore each app needs to be tested together, trusted, and a compatible [Content-Security-Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) should be considered for each of them.
```
Comment (joaquimrocha) [inline docs/installation/base-url.md:15]:
```suggestion
Hosting multiple websites (apps) on the [same origin](https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy) can lead to possible conflicts between the apps. Each app is able to access information and make requests of the other. Therefore each app needs to be **tested** together, **trusted**, and a compatible **[Content-Security-Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP)** should be considered for each of them.
```

File: docs/installation/base-url.md [hunk 2]
```diff
@@ -6,8 +6,15 @@ linkTitle: Base URL
...
+
+If in doubt, host headlamp on a separate origin (domain or port).
```
Comment (joaquimrocha) [inline docs/installation/base-url.md:17]:
```suggestion
If in doubt, host Headlamp on a separate origin (domain or port, don't use the `-base-url` option).
```

**Review by joaquimrocha — CHANGES_REQUESTED** [review]
Left some suggestions.

**Comment by illume** [comment]
Thanks. I squished in your suggestions.

**Review by joaquimrocha — APPROVED** [review]

**Pull request merged**